rdbms/src/main/java/org/spine3/server/storage/jdbc/util/DataSourceWrapper.java

```diff
@@ -30,17 +30,24 @@
 import java.sql.SQLException;
 
 /**
- * The base class for {@link DataSource} wrappers.
+ * Wrapper for {@link DataSource}.
```

Comment:
We can use no article if we mean `any`. In this case, it is better to use `a` or `the`.